lib/tvdbapiv2/api_client.py

```diff
@@ -207,10 +208,14 @@ def deserialize(self, response, response_type):
             return self.__deserialize_file(response)
 
         # fetch data from response object
+        if six.PY3:
```

Comment:
It should decode in both py2 and py3.

Reply:
I will change it when I rebase all the commits. Thank you!